Further usecases (#13)
ElektraInitiative/PermaplanT: doc/usecases/layers/trees_layer.md

# Use Case: Trees Layer

## Summary

- **Scope:** Trees Layer
- **Level:** User Goal
- **Actors:** App User

> - **Brief:** The user can add, edit, move and delete trees and other permanent structures such as bushes in their plan.

Reviewer: ```suggestion
- **Brief:** The user can add, edit, move and delete forest, trees and, hedge bushes in their map.
```

- **Status:** Draft

## Scenarios

- **Precondition:**
  The user has opened the app and has selected the trees layer.
- **Main success scenario:**

>   The user successfully adds, edits, moves and deletes trees and other permanent structures in their landscape plan using the trees layer. This includes positioning the stems, adjusting the height, sketching the shape, and specifying the type of tree or structure.

Reviewer: Please put new sentences in new lines. Fix as corrected above.
Forest is an area of trees (individual trees not drawn), hedge an area of bushes.
